velox/exec/Task.cpp

```diff
@@ -935,6 +948,31 @@ void Task::addHashJoinBridgesLocked(
   }
 }
 
+void Task::addCustomizedJoinBridgesLocked(
+    uint32_t splitGroupId,
+    const std::vector<std::shared_ptr<const core::PlanNode>> planNodes) {
+  auto& translators = Task::translators();
+  if (0 == translators.size()) {
+    return;
+  }
+
+  auto& splitGroupState = splitGroupStates_[splitGroupId];
+  for (const auto& planNode : planNodes) {
+    for (const auto& t : translators) {
+      if (auto joinBridge = t->translate(planNode)) {
+        splitGroupState.bridges.emplace(planNode->id(), joinBridge);
```

Comment:
Should we continue to the next plan node once we found a bridge?

Reply:
Just return when we found a bridge